web/src/pages/about.tsx

```diff
@@ -0,0 +1,123 @@
+    img: hannahWragg,
+  },
+  {
+    id: "01GCT0D1WWYGATDR1J71WSZEJB",
+    name: "Rahul Shah",
+    role: "Community and Logistics",
+    bio: "Rahul's a medical student studying at Cambridge, and has been involved in Raise since the start of his time there: first as a rep, then as Vice President Fundraising, then as President. He's excited to join the National Team to help Raise expand, evolve and continue having huge impact.\n\nWhen not doing Raise stuff, Rahul likes taking on entrepreneurial side projects and attempting to cook!",
+    img: rahulShah,
+  },
+  {
+    id: "01GCT0D1WWYDDHKDY59H0DAQ5C",
+    name: "Susanne Karbe",
+    role: "Community",
+    bio: "Susanne recently completed her PhD in Economics at Durham University and is currently on a gap year in South Korea! She set up the Raise chapter in Durham in 2020 and is now on the national team.\n\nIn her free time she loves to travel, dance and skate!",
+    img: susanneKarbe,
+  },
+  {
+    id: "01GCT0D1WW3E0YN30JKEQEPJY1",
+    name: "Thomas Cohen",
+    role: "Treasurer and Logistics",
+    bio: "Thomas does a number of different jobs on the Raise National Team, including managing the accounts and mentoring. He was heavily involved in Raise in Durham, acting as Vice President and President over its first two years. He is really passionate about positive and celebratory giving, and therefore has pledged to give 10% of his income to charity!\n\nThomas has just finished a degree in History at Durham, and this year will be doing a master's in History and Philosophy of Science at UCL.",
+    img: thomasCohen,
+  },
+]
+
+const AboutPage = () => (
+  <Page>
+    <Helmet>
+      <title>Raise: About Us</title>
+      <meta property="og:title" content="Raise: About Us" />
+    </Helmet>
+    <Navigation
+      left={[
+        { text: "Alumni", href: "/alumni/" },
+        { text: "Bristol", href: "/bristol/" },
+        { text: "Cambridge", href: "https://www.mayweekalternative.org.uk/" },
+        { text: "Durham", href: "/durham/" },
+        { text: "Edinburgh", href: "/edinburgh/" },
+        { text: "Glasgow", href: "/glasgow/" },
+        { text: "Oxford", href: "/oxford/" },
+        { text: "Sheffield", href: "/sheffield/" },
+        { text: "Warwick", href: "/warwick/" },
+        { text: "About", href: "/about/" },
```

Comment:
Will edit the navigation on the other pages once we're happy internally with making this 'more' public.